oemoflex/model/facade_attrs/link.csv

```diff
@@ -7,5 +7,6 @@ tech,str,n/a,Technology
 from_bus,str,n/a,Bus where link starts
 to_bus,str,n/a,Bus where link ends
 capacity,float,MW,Net transfer capacity
```

Comment:
The attribute `capacity` in oemof.tabular's Link class has been replaced by `from_to_capacity`, `to_from_capacity`, which allows to set different values for different directions. 
```suggestion
from_to_capacity,float,MW,Net transfer capacity in defined direction
to_from_capacity,float,MW,Net transfer capacity opposite to defined direction
```

Reply:
Affects test data, too.